modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -739,6 +739,11 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<name>sortItems</name>
```

Comment:
sortList?

Reply:
As the one before, I was watching "bulkActionDropdownItems", and set the new one alike.